### PR TITLE
Images and Input

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Input and Images
Input include: email, Username, Password and Text box.
Images: Responsive Image, round responsive image.

### CODE REVIEW

File: styles/styles-global/root.css [hunk 1]
```diff
@@ -32,4 +33,12 @@ body{
         "nav  foot";
 grid-template-rows: 50px 1fr 30px;
 grid-template-columns: 200px 1fr;
+}
+
+.container{
+    border: 5px solid var(--dark-grey-color) !important;
```
Comment (mehulsatardekar) [inline styles/styles-global/root.css]: Avoid using !important. instead, you can increase the specificity here.

https://stackoverflow.com/questions/27443589/how-do-i-avoid-using-important#:~:text=To%20avoid%20using%20!,%22%2C%20and%20thus%20getting%20overridden.

Comment (mehulsatardekar) [reply styles/styles-global/root.css]: looks good to merge after the changes 🚀

Comment (vanshitamoolchandani) [reply styles/styles-global/root.css]: Thank you for the suggestion.

File: component/Images/styles.css [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+
+.img-responsive{
+    max-inline-size: 100%;
+    block-size: auto;
+    object-fit: contain;
+    
+}
+
+.img-rect{
+    /* aspect-ratio: 1/1; */
```
Comment (rahulk31) [inline component/Images/styles.css:56]: can remove the commented code.

**Review by AshutoshM29 — COMMENTED** [review]
Looks good to me!
You can merge it!

**Review by rahulk31 — APPROVED** [review]
Could not find any other flaw. Can be merged Vanshita 🚀

**Pull request merged**